### DISCUSSION TITLE
After Github Education Error: We Are Having a Problem Billing Your Account.

### DISCUSSION THREAD

**Discussion by dusova** [community/community#146409] [GitHub Education] [answered]
### Hi everyone,

![image](https://github.com/user-attachments/assets/ba12dbb0-b752-4761-b463-4264ae0c68af)
I have a error after join Github Education Student. How can i fix it?

**Comment by davevad93** [comment, marked as answer]
Hi @dusova ,

In this case I recommend you to contact support by opening a ticket [here](https://support.github.com/contact/account), select “`Billing, payments, or receipts`” and explain your issue.

**Reply by dusova** [reply]
I will try this solution. Thank you so much :)

**Reply by davevad93** [reply]
You're welcome @dusova 👍
